library/src/main/java/com/novoda/downloadmanager/MigrationExtractor.java

```diff
@@ -7,14 +7,16 @@
 
 class MigrationExtractor {
 
-    private static final String BATCHES_QUERY = "SELECT DISTINCT batches._id, batches.batch_title FROM batches INNER JOIN DownloadsByBatch ON "
-            + "DownloadsByBatch.batch_id = batches._id WHERE DownloadsByBatch.batch_total_bytes = DownloadsByBatch.batch_current_bytes";
-    private static final String URIS_QUERY = "SELECT uri, _data, total_bytes FROM Downloads WHERE batch_id = ?";
+    private static final String BATCHES_QUERY = "SELECT batches._id, batches.batch_title FROM batches INNER JOIN DownloadsByBatch ON "
```

Comment:
don't we need the `DISTINCT` anymore?

Reply:
"GROUP BY batches._id" achieves the same thing and a little more obvious 😄